infra/helper.py

```diff
@@ -817,6 +819,11 @@ def _get_latest_corpus(project, fuzz_target, base_corpus_dir):
   if not fuzz_target.startswith(project.name + '_'):
     fuzz_target = '%s_%s' % (project.name, fuzz_target)
 
+  # Replace chars from fuzztest fuzzers that are replaced in coverage URLs.
+  for c in FUZZTEST_CORPUS_URL_REPLACE_CHARS:
```

Comment:
Can you use this regex instead:

https://github.com/google/clusterfuzz/blob/ead341fec9eeea5dfb84b10cde1601a270838ea8/src/clusterfuzz/_internal/datastore/data_types.py#LL1225C18-L1225C18

https://github.com/google/clusterfuzz/blob/ead341fec9eeea5dfb84b10cde1601a270838ea8/src/clusterfuzz/_internal/datastore/data_types.py#LL84-L84C20

Reply:
Fixed, copied over the exact logic